README.md

```diff
@@ -1,12 +1,37 @@
-## game plan
-### cc binary that displays leaderboard
-```sh
-sudo ./text-example \
-    --led-rows=64 \
-    --led-cols=64 \
-    --led-chain=4 \
-    --led-gpio-mapping=adafruit-hat \
-    --led-slowdown-gpio=4 \
-    --led-pixel-mapper="U-mapper" \
-    --led-slowdown-us=10
-```
+
+# SCE LED Matrix
+
+The SCE LED Matrix is an SCE intern project for Winter 2024. The matrix consists of four interconnected LED panels, controlled by a Raspberry Pi.  
+This repository contains code and resources for controlling and managing the SCE LED Matrix Display. The below is a description of the layout of this repository.
+
```

Comment:
this readme should only be be

1. (this can be in a later pr when we add the c++ code) how to compile the sign binary, where does the rpi-rgb-led-matrix be cloned relative to the c++ code?
2. how to run the server
3. explain how we store snapshots, also how a score is calcualted